core-bundle/src/Security/Authentication/ContaoLoginAuthenticationListener.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Security\Authentication;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Http\Firewall\UsernamePasswordFormAuthenticationListener;
+
+class ContaoLoginAuthenticationListener extends UsernamePasswordFormAuthenticationListener
```

Comment:
Why did you prefix the class name with `Contao`? It should be `LoginAuthenticationListener` IMHO.

Reply:
Because the firewall listener/factory is called `contao_login`